test/Microsoft.ML.Tests/ImagesTests.cs

```diff
@@ -262,7 +262,7 @@ public void TestBackAndForthConversionWithoutAlphaInterleave()
             var pixels = new ImagePixelExtractingTransformer(env, "ImagePixels", "ImageCropped", interleave: true, scale: 2f / 19, offset: 30).Transform(cropped);
```

Comment:
>ImagePixelExtractingTransformer [](start = 29, length = 31)

Right now ImagePixel and VectorToImage are pair of opposite to each over transforms. it's weird to have in one of them `interleave` parameter and `interleavedColors` in another.

I know both this constructors are internal, but it's still looks weird.

Reply:
In the public constructors and public api, I intentionally named the parameter differently because the two transforms are doing the opposite of each other. In ImagePixel, the parameter specifies whether to **interleave** the colors or not when producing the vector, so I named it `interleavePixelColors`. In VectorToImage, the parameter specifies whether the colors in the vector are **interleaved** or not, so I named it `interleavedColors`. The intention was to make the parameter names descriptive, like all the other parameters that I renamed.

Do you agree with this distinction in names, i.e. `interleavePixelColors` for ImagePixel and `interleavedColors` for VectorToImage? If so, I can rename the parameters in the internal constructors to be consistent with the public ones. If not, do you suggest I use the same name for both ImagePixel and VectorToImage and what should that name be? In either case, I agree the names in public and internal constructors should be consistent.

Reply:
Also, is it fine to make the rest of the parameters descriptive as I have done or do you prefer the previous short names e.g. `offset` (old) vs `offsetPixelColor` (new) ?

@Ivanidzo4ka